default.nix

```diff
@@ -73,6 +73,7 @@ let
     inherit (jmPkgs) jormungandr jormungandr-cli;
     inherit (pkgs) cardano-node;
     inherit (haskellPackages.cardano-wallet-core.identifier) version;
+    inherit (haskellPackages.ouroboros-consensus-byron.components.exes) db-converter;
```

Comment:
Could you add a comment explaining why this needs to be here? Otherwise someone might remove it.